### ISSUE TITLE
What does a yaml2po po2yaml template look like?

### ISSUE THREAD

**Issue by boydkelly** [translate/translate#4512]
Both these utilities give output stating the requirement for a yaml template.  However there does not seem to be any example anywhere as to what that template should look like?   Does it have keys msgstr?   Does it have keys representing the string to be translated? Does it have language symbols?   Is it one template per language or multiple languages?   Do you *have* to start with a yaml file to po?

**Comment by guillaumechereau** [comment]
I am also looking for this information.

**Comment by nijel** [comment]
It's the yaml file you are translating (used to generate the PO file using yaml2po).

**Comment by guillaumechereau** [comment]
Thanks. I think I got confused about what the template is. Is there a way to create a yaml from a po file that wasn't originally generated with yaml2po?

**Comment by nijel** [comment]
No, this is really intended for generating translated YAML files. You start with yaml2po, translate PO files and then translate the original YAML file with po2yaml using it as tempalte.
